apps/zui/src/app/core/models/zed-ast.ts

```diff
@@ -34,6 +23,12 @@ export class ZedAst {
     return trunks.filter((t) => t.source.kind === "Pool").map((t) => t.source)
   }
 
+  get groupByKeys() {
+    const g = this.ops.find((op) => op.kind === "Summarize")
+    const keys = g ? g.keys : []
+    return keys.map((k) => fieldExprToName(k.lhs || k.rhs))
+  }
```

Comment:
Just to capture something I think was discussed on Zoom, at some point in the future we'd hope that we could rely on the Zed backend to start providing this kind of summary hinting rather than the app having to pick apart the AST to make its own conclusions. @jameskerr is that accurate?

Reply:
Yes that is correct.